src/common/services/ArtemisService.ts

```diff
@@ -25,7 +26,30 @@ export class ArtemisService {
         const { geoName, environment, clusterNumber } = artemisResponse.response as IArtemisAPIOrgResponse;
         sendTelemetryEvent({ eventName: CopilotArtemisSuccessEvent, copilotSessionId: sessionID, geoName: String(geoName), orgId: orgId });
 
-        const crossGeoDataMovementEnabledPPACFlag = await BAPService.getCrossGeoCopilotDataMovementEnabledFlag(artemisResponse.stamp,environmentId);
+        // Check if governance FCB is enabled
+        const isGovernanceCheckEnabled = isCopilotGovernanceCheckEnabled();
+
+        if (isGovernanceCheckEnabled) {
+
+            const copilotGovernanceSetting = getCopilotGovernanceSetting();
+
+            sendTelemetryEvent({ eventName: CopilotGovernanceCheckEnabled, copilotSessionId: sessionID, orgId: orgId });
```

Comment:
Should we also log `isGovernanceCheckEnabled` and `copilotGovernanceSetting` here?